mods/cnc/weapons/missiles.yaml

```diff
@@ -298,94 +209,53 @@ TowerMissle:
 		Versus:
 			None: 50
 			Wood: 25
-			Light: 100
-			Heavy: 100
 		DamageTypes: Prone50Percent, TriggerProne, SmallExplosionDeath
-	Warhead@2Smu: LeaveSmudge
-		SmudgeType: Crater
-		InvalidTargets: Vehicle, Structure, Wall, Husk, Trees, Creep
 	Warhead@3Eff: CreateEffect
 		Explosions: med_frag
 		ImpactSounds: xplos.aud
 
 SAMMissile:
+	Inherits: ^MissileWeapon
 	ReloadDelay: 15
 	Range: 8c0
 	Report: rocket2.aud
 	ValidTargets: Air
 	Projectile: Missile
-		Arm: 0
-		Blockable: false
 		Image: MISSILE
 		HorizontalRateOfTurn: 20
 		Speed: 426
 		RangeLimit: 9c614
-		TrailImage: smokey
-		ContrailLength: 8
 	Warhead@1Dam: SpreadDamage
 		Spread: 682
+		Damage: 35
 		ValidTargets: Air
 		Versus:
 			None: 100
 			Wood: 100
 			Light: 100
 			Heavy: 75
-		Damage: 35
-		DamageTypes: Prone50Percent, TriggerProne, ExplosionDeath
 	Warhead@3Eff: CreateEffect
 		Explosions: small_building
 		ImpactSounds: xplos.aud
 
-HonestJohn:
-	ReloadDelay: 200
-	Range: 10c0
-	Report: rocket1.aud
-	Projectile: Bullet
-		Arm: 10
-		Blockable: false
-		Shadow: true
-		Inaccuracy: 213
-		Image: patriot
-		TrailImage: smokey
-		Speed: 187
-		RangeLimit: 35
-		LaunchAngle: 88
-	Warhead@1Dam: SpreadDamage
-		Spread: 256
-		Damage: 100
-		Versus:
-			None: 90
-			Wood: 75
-			Light: 60
-			Heavy: 25
-		DamageTypes: Prone50Percent, TriggerProne, SmallExplosionDeath
-	Warhead@2Smu: LeaveSmudge
-		SmudgeType: Crater
-	Warhead@3Eff: CreateEffect
-		Explosions: small_poof
-		ImpactSounds: xplos.aud
-
 Patriot:
+	Inherits: ^MissileWeapon
```

Comment:
The versus changes were 'intentional' (to save some yaml lines), because all targetable aircraft use Light armor.